src/views/Books/BookDetail.js

```diff
@@ -1,9 +1,11 @@
 import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import Book from '../../components/book/Book';
 import { getBookById } from '../../services/books';
 
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const params = useParams();
+  const id = params.id; // TODO: Use id from route
```

Comment:
This can also be written as a one liner => { id } = useParams() where we destructure the id